src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/Internal/TemplateBuilder.cs

```diff
@@ -72,7 +72,7 @@ public string Build()
             // We need to copy the ModelExplorer to copy the model metadata. Otherwise we might
             // lose track of the model type/property. Passing null here explicitly, because
             // this might be a typed VDD, and the model value might not be compatible.
```

Comment:
Add a bit to this comment about why the next line creates a `ViewDataDictionary<object>`.